app/routes/court-reports-overview.js

```diff
@@ -0,0 +1,36 @@
+const getCourtReportOverview = require('../services/get-court-report-overview')
+const getSubNav = require('../services/get-sub-nav')
+const getOrganisationUnit = require('../services/helpers/org-unit-finder')
+const organisationUnitConstants = require('../constants/organisation-unit')
+const workloadTypeConstants = require('../constants/workload-type')
+
+module.exports = function (router) {
```

Comment:
I think that the court-reporter overview page is different enough to deserve its own route.  We use a different db view, and unlike the probation overview we have a single view for all rather than an individual view and an organisation view